### DISCUSSION TITLE
체크아웃이 활성화 되지 않습니다.

### DISCUSSION THREAD

**Discussion by LEEARA304** [woocommerce/woocommerce#50955] [Checkout Flow]
체크아웃에 고유통관부호 번호를 넣고 싶은데 어떻게 해야할지를 모르겠습니다.

**Comment by senadir** [comment]
Hey! It's not possible to add fields using the UI right now but you can use simple code to add a field to Checkout

https://developer.woocommerce.com/docs/cart-and-checkout-additional-checkout-fields/
